tests/jerry/es2015/number-isinteger.js

```diff
@@ -0,0 +1,21 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+assert(Number['isInteger'] ('valami') === false);
+assert(Number['isInteger'] (NaN) === false);
+assert(Number['isInteger'] (Infinity) === false);
+assert(Number['isInteger'] (-Infinity) === false);
+assert(Number['isInteger'] (0) === true);
+assert(Number['isInteger'] (-100000) === true);
+assert(Number['isInteger'] (5.0000000000000001) === true);
```

Comment:
There should be a test case where the argument is a number and the expected result is `false`.
like: `Number.isInteger(3.5) === false`